config.go

```diff
@@ -794,6 +794,12 @@ func (c *Config) Validate() error {
 		return ConfigurationError("ReadCommitted requires Version >= V0_11_0_0")
 	}
 
+	// backward compatibility on Rebalance Strategy
```

Comment:
The code works, but I'm not sure if it's good to modify the config in `Validation` method. 

And btw there is no need to clear `c.Consumer.Group.Rebalance.GroupStrategies` here.
Because `c.Consumer.Group.Rebalance.Strategy` takes precedence of `c.Consumer.Group.Rebalance.GroupStrategies` in sarama (
https://github.com/Shopify/sarama/blob/v1.37.0/consumer_group.go#L443-L453 & https://github.com/Shopify/sarama/blob/v1.37.0/consumer_group.go#L333-L340)